File: backend/services/competency-service/src/main/java/com/lmspilot/competency/CompetencyServiceApplication.java
package com.lmspilot.competency;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;


@SpringBootApplication(scanBasePackages = "com.lmspilot")
public class CompetencyServiceApplication {
    public static void main(String[] args) { SpringApplication.run(CompetencyServiceApplication.class, args); }
}

File: backend/services/competency-service/src/main/kotlin/com/lmspilot/competency/CompetencyServiceApplication.kt
package com.lmspilot.competency

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.lmspilot"])
class CompetencyServiceApplication

fun main(args: Array<String>) {
    runApplication<CompetencyServiceApplication>(*args)
}
